src/es/jkanime/src/eu/kanade/tachiyomi/animeextension/es/jkanime/extractors/JkanimeExtractor.kt

```diff
@@ -42,4 +42,14 @@ class JkanimeExtractor(
         }
         return null
     }
+
+    fun getDesukaFromUrl(url: String, prefix: String = ""): Video? {
```

Comment:
Since you're using `!!` after calling the function anyways, make it return `List<Video>` instead and return an emptyList instead of null